Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -225,7 +220,7 @@
 @property (nonatomic, copy, readonly) NSString *sourceName;
 
 /**
- *  The unique identifier for the source: the channelId
+ *  The unique identifier for the source: the appId
  */
 @property (nonatomic, copy, readonly) NSString *sourceIdentifier;
```

Comment:
@maximebedard when using app extensions, is using the `app ID` as the `source_identifier` correct?